src/Microsoft.AspNetCore.Http.Connections/ConnectionsAppBuilderExtensions.cs

```diff
@@ -9,8 +9,17 @@
 
 namespace Microsoft.AspNetCore.Builder
 {
+    /// <summary>
+    /// Extension methods for <see cref="IApplicationBuilder"/>.
+    /// </summary>
     public static class ConnectionsAppBuilderExtensions
     {
+        /// <summary>
+        /// Adds SignalR HTTP connectivity to the <see cref="IApplicationBuilder"/> request execution pipeline.
```

Comment:
This layer isn't SignalR though (although I'm not sure that nuance is something users care about...).

How about: "Adds support for ASP.NET Core Connection Handlers to the [IApplicationBuilder] request execution pipeline"